lib/Target/TTNN/TTNNToFlatbuffer.cpp

```diff
@@ -683,6 +683,48 @@ createOp(FlatbufferObjectCache &cache, MorehCumSumOp op) {
                                                  op.getDim(), memoryConfig);
 }
 
+::flatbuffers::Offset<::tt::target::ttnn::PrepareConv2dWeightsOp>
+createOp(FlatbufferObjectCache &cache, PrepareConv2dWeightsOp op) {
+  auto weight_tensor = cache.at<::tt::target::ttnn::TensorRef>(
+      getOperandThroughDPSOps(op.getWeightTensor()));
+  auto output = cache.getOrCreate(op.getResult(), tensorValueToFlatbuffer,
+                                  kHostAllocatedSize);
+
+  ::flatbuffers::Offset<::tt::target::ttnn::MemoryConfig> memoryConfig =
+      memoryConfigToFlatbuffer(
+          cache, op.getInputMemoryConfig(),
+          getTensorValueTileShape(op.getResult()),
+          getTensorValueCoreRangeSet(cache, op.getResult()));
+
+  ::tt::target::TensorLayout input_tensor_layout =
+      toFlatbuffer(cache, op.getInputTensorLayout());
+  ::flatbuffers::Offset<::flatbuffers::String> weights_format =
+      toFlatbuffer(cache, op.getWeightsFormat());
+
+  ::flatbuffers::Offset<::flatbuffers::Vector<int32_t>> kernelSize =
+      toFlatbuffer(cache, op.getKernelSize());
+  ::flatbuffers::Offset<::flatbuffers::Vector<int32_t>> stride =
+      toFlatbuffer(cache, op.getStride());
+  ::flatbuffers::Offset<::flatbuffers::Vector<int32_t>> padding =
+      toFlatbuffer(cache, op.getPadding());
+  ::flatbuffers::Offset<::flatbuffers::Vector<int32_t>> dilation =
+      toFlatbuffer(cache, op.getDilation());
+  auto device = getOperandThroughDPSOps(op.getDevice());
+
+  std::optional<::flatbuffers::Offset<::tt::target::ttnn::Conv2dConfig>>
+      conv2dConfig;
+  if (auto config = op.getConv2dConfig()) {
+    conv2dConfig = toFlatbuffer(cache, *config);
+  }
```

Comment:
You can rebase on the latest main and use:
```suggestion
std::optional<::flatbuffers::Offset<::tt::target::ttnn::Conv2dConfig>>
      conv2dConfig = toFlatubffer(cache, op.getConv2dConfig());
```